README.md

```diff
@@ -3,5 +3,123 @@ Babbage
+publish_date_too_far_in_past_total 0.0
+# HELP publish_date_too_far_in_future_total Total requests for uris that have a future publishing date later than that calculated by the default expiry time
+# TYPE publish_date_too_far_in_future_total counter
+publish_date_too_far_in_future_total 0.0
+# HELP publish_date_not_present_total Total requests for uris that have no publishing date found
+# TYPE publish_date_not_present_total counter
+publish_date_not_present_total 0.0
+# HELP publish_date_in_range_total Total requests for uris that have a publishing date within the range required for setting the cache expiry time
+# TYPE publish_date_in_range_total counter
+publish_date_in_range_total 0.0
+# HELP publish_date_in_range_created Total requests for uris that have a publishing date within the range required for setting the cache expiry time
+# TYPE publish_date_in_range_created gauge
+publish_date_in_range_created 1.686667997375E9
+# HELP publish_date_not_present_created Total requests for uris that have no publishing date found
+# TYPE publish_date_not_present_created gauge
+publish_date_not_present_created 1.686667997376E9
+# HELP publish_date_too_far_in_future_created Total requests for uris that have a future publishing date later than that calculated by the default expiry time
+# TYPE publish_date_too_far_in_future_created gauge
+publish_date_too_far_in_future_created 1.686667997376E9
+# HELP publish_date_too_far_in_past_created Total requests for uris that have a past publishing date too long ago (outside a given time span)
+# TYPE publish_date_too_far_in_past_created gauge
+publish_date_too_far_in_past_created 1.686667997376E9
+```
+
+### Debugging
+
+When Babbage is run using one of its scripts (either run.sh or run-publishing.sh) it incorporates a Java Debug Wire Protocol (JDWP).
+
+To create the configuration in Intellij, for calling the JDWP debugger, do as follows:
+
+- Choose Run --> Edit Configurations...
+- Click the + to create a new configuration
+- For the type of configuration select either 'Remote' or 'Remote JVM' (whichever option it gives you)
+- Give the new configuration a name e.g. Babbage Remote
+- For 'Debugger mode' choose 'Attach to remote JVM'
+- For 'Host' enter: localhost
+- For 'Port' enter the port number given in the relevant script (e.g. for run-publishing.sh it's 8000)
+- Intellij should automatically complete the command line arguments (note that these are similar to the jdwp arguments in the relevant script):
+
+  -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8000
+
+Then, to run and debug Babbage just do the following:
+
+- At the command line, run the relevant script E.g.,
+cd babbage
+./run-publishing.sh
+
+- Then in Intellij:
+- Open babbage and add any breakpoints required
+- Choose Run --> Debug 'Babbage Remote'
+
+### Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md) for details.
+
+### License
+
+Copyright © 2022, Office for National Statistics (https://www.ons.gov.uk)
+
```

Comment:
2023?

Reply:
I asked about this at stand up this morning. I was told that it's probably best to remove the copyright notice for now because @s-xton is currently seeking clarity on the situation with it. So I shall take it out for now, thanks.